src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -143,7 +144,10 @@ export default class ClinicalData extends React.Component<
                 );
             } else {
                 content.push(
-                    <OverlapExclusionIndicator store={this.props.store} />
+                    <div className={'tabMessageContainer'}>
```

Comment:
needed this class to get proper margin around the messages

Reply:
that class was meant for a particular spot on the page so might be problematic here in future.